public/app/features/datasources/settings/CloudInfoBox.tsx

```diff
@@ -38,27 +39,38 @@ export const CloudInfoBox: FC<Props> = ({ dataSource }) => {
   }
 
   return (
-    <DismissableFeatureInfoBox
-      title={`Configure your ${mainDS} data source below`}
-      persistenceId="data-source-settings-cloud-info-box"
-      className={styles.box}
-      branded={false}
-    >
-      <div className={styles.text}>
-        Or skip the effort and get {mainDS} (and {extraDS}) as fully managed, scalable and hosted data sources from
-        Grafana Labs with the{' '}
-        <a
-          className="external-link"
-          href={`https://grafana.com/signup/cloud/connect-account?src=grafana-oss&cnt=${dataSource.type}-settings`}
-          target="_blank"
-          rel="noreferrer"
-          title="The free plan includes 10k active metrics and 50gb storage."
-        >
-          free-forever Grafana Cloud plan
-        </a>
-        .
-      </div>
-    </DismissableFeatureInfoBox>
+    <LocalStorageValueProvider<boolean> storageKey="datasources.settings.cloudInfoBox.isDismissed" defaultValue={false}>
```

Comment:
Nit: maybe we could make it a constant, to keep things consistent with other usages, i.e. https://github.com/grafana/grafana/blob/1d689888b0fc2de2dbed6e606eee19561a3ef006/public/app/core/components/TimePicker/TimePickerWithHistory.tsx#L12